test/testdata/array-errors.json

```diff
@@ -3,7 +3,7 @@
 		"lineNumber": "1",
 		"properties": {
 			"recordID": "1",
-			"filler": " "
+			"filler": ""
```

Comment:
Ok. just looked a little odd alongside the rest of the commits and wanted to make sure